src/main/java/net/sacredlabyrinth/phaed/simpleclans/ui/frames/Components.java

```diff
@@ -103,7 +106,7 @@ public static SCComponent getClanComponent(@NotNull SCFrame frame, @NotNull Play
             lore = Arrays.asList(
                     lang("gui.clandetails.clan.lore.description", viewer,
                             clan.getDescription() != null && !clan.getDescription().isEmpty() ? clan.getDescription() : lang("no.description", viewer)),
-                    lang("gui.clandetails.clan.lore.status", viewer, clan.isVerified() ? lang("verified", viewer) : lang("unverified", viewer)),
+                    lang("gui.clandetails.clan.lore.status", viewer, Stream.of(clan.isVerified() ? lang("verified", viewer) : lang("unverified", viewer), clan.isPermanent() ? lang("permanent", viewer) : null).filter(Objects::nonNull).collect(Collectors.joining(", "))),
```

Comment:
I think, there should be a method in Clan: `clan.getStatuses()`.
What do you think?

Reply:
Maybe in Helper